src/base/bittorrent/session.cpp

```diff
@@ -1796,10 +1800,14 @@ void Session::processShareLimits()
                             logger->addMessage(tr("'%1' reached the maximum seeding time you set. Removed.").arg(torrent->name()));
                             deleteTorrent(torrent->hash());
                         }
-                        else if (!torrent->isPaused()) {
+                        else if ((m_maxRatioAction == Pause) && !torrent->isPaused() ) {
```

Comment:
Redundant space after isPaused()

Reply:
This method has the same code in 2 places (for max ratio and for max seeding time). Maybe to rearrange it?

Reply:
>Maybe to rearrange it?

I'd rather not.